### PR TITLE
deleted home-template and make this repo of sole vue

### PR DESCRIPTION
**Pull request by khushalbhardwaj-0111**

### CODE REVIEW

**Review by khushalbhardwaj-0111 — COMMENTED** [review]
merging the pr

File: src/components/css/main.css [hunk 1]
```diff
@@ -0,0 +1,181 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans&family=Raleway&family=Roboto+Slab&display=swap");
```
Comment (justtheory) [inline src/components/css/main.css:1]: please move the styles to respective component
use style scoped to make styles applicable only for those templates

File: package.json [hunk 2]
```diff
@@ -8,7 +8,6 @@
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
     "start": "node ./backend/index.js"
```
Comment (justtheory) [inline package.json:10]: this command needs to be removed or updated as we don't have backend in this repo

File: src/components/todo-list-item.vue [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<template>
```
Comment (justtheory) [inline src/components/todo-list-item.vue:1]: please complete the design changes required

File: package.json [hunk 4]
```diff
@@ -22,7 +21,9 @@
     "@vue/compiler-sfc": "^3.0.0",
     "babel-eslint": "^10.1.0",
     "eslint": "^6.7.2",
```
Comment (justtheory) [inline package.json:23]: why do we have multiple lint packages

**Review by justtheory — COMMENTED** [review]
Rename first character of all component such that they begin with capital letter

**Comment by khushalbhardwaj-0111** [comment]
Closing this to commit new changes

**Pull request closed without merging**